content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+
+While there are various reasons a user might not be able to see weakly contrasted color, everyone is different. See if you can distinguish the text from the background in the displayed image with poor contrast:
+
+<img alt="Dark gray text on a black background" src="./color_fail.png" style="max-width: 600px; width: 100%"/>
+
+Now, compare that to highly contrasting colors: 
+
+<img alt="White text on a black background" src="./color_pass.png" style="max-width: 600px; width: 100%"/>
+
+The contents are not only easier to distinguish from the background, but it's made easier to focus on as a result of the increased contrast.
+
+This said, not all contrasts are the same. Per [WCAG guidelines](#wcag), you may have a different ratio of contrast for different compliance levels. These contrast ratios depend on both font size as well as compliance level.
+
+![Light grey text passing AA for large text, but not small text](./pass_aa_large.png)
+
+In this example you can see that the text passes the WCAG AA requirements for large text, but fails the same requirements for small text.
+
+# Fantastic Fonts {#font-resize}
+
+One of the most widely used accessibility features is font scaling. While many browsers default to a font size of `16px`, the user is actually able to change settings on their device to configure websites to use a larger font size.
+
+Many phones using iOS and Android allow users to change the font size on their mobile devices. This feature is heyso commonplace that it often prompts users to change this setting when the phone is being set up.
+
+<div style="display: flex; justify-content: space-around">
+    <figure>
+		<img src="./ios_text_size.png" style="height: 300px" alt=""/>
+        <figcaption>
+            iOS font size settings screen
+        </figcaption>
+    </figure>
+    <figure>
+        <img src="./android_text_size.png" style="height: 300px" alt=""/>
+        <figcaption>
+            Android font size settings screen
+        </figcaption>
+    </figure>
+</div>
+
+Not only do you have these settings on mobile devices, but they're available on desktop as well. 
+
+Using Chrome, go to [your settings page](chrome://settings/?search=font+size), and you should be able to set your font size.
+
+![Font settings in Chrome](./chrome_font_size.png)
+
+You can do the same in Firefox in [your preferences](about:preferences#general).
+
+![Font settings in Firefox](./firefox_font_size.png)
+
+
+## Implementation {#font-rem}
+
+While browsers have the ability to set the font size, if you're using `px`, `vw`, `vh`, or other unit values for your fonts, the browser will not update these font sizes for you. In order to have your application rescale the font size to match the browser settings, you'll need to use the `rem` unit.
```

Comment:
`em` is a critical length unit relative to the parent element. Is it possible to add this to this section?

Reply:
I understand `em`'s usage, but don't understand its usage in context to A11Y - could you teach me? How do you see it being mentioned/added? (don't mind if you wanna do over DMs, since GH issues is far from expedient)

Reply:
`em`s are used similarly to `rem`s, so I want to make sure `you'll need to use the `rem` unit` doesn't imply that em is off the table. Not sure the exact solution or if that's a tangent though

Reply:
The only reason I tend to avoid mentioning `em` is because they _are_ relative to the parent. Because of this, teams looking to do incremental migration away from `px` to properly sizing fonts may pick `em` and not realize that this doesn't fix the issue. I personally prefer to default to `rem` anyway, then use `em` when contextually relevant

At either rate, I think that `rem` acts as a decent default with less nuance required than `em`, and explaining further might be out of context for this particular article